Makefile

```diff
@@ -71,15 +71,15 @@ notes: check-bump
 	# Now generate the release notes to have them included in the release commit
 	towncrier build --yes --version $(UPCOMING_VERSION)
 	# Before we bump the version, make sure that the towncrier-generated docs will build
-	make build-docs
+	make check-docs
 	git commit -m "Compile release notes"
 
 release: check-bump clean
 	# require that you be on a branch that's linked to upstream/master
 	git status -s -b | head -1 | grep "\.\.upstream/master"
 	# verify that docs build correctly
 	./newsfragments/validate_files.py is-empty
-	make build-docs
+	make check-docs
```

Comment:
Same thing here, the preview will be empty, and there's already an explicit validation that the newsfragments are empty in the line above. (The `is-empty` check could be copied to the `make notes` section, above where the `make build-docs` was.)